iotdb-core/consensus/src/main/java/org/apache/iotdb/consensus/ratis/RatisConsensus.java

```diff
@@ -869,9 +873,34 @@ private RaftClientReply sendReconfiguration(RaftGroup newGroupConf)
     // notify the group leader of configuration change
     RaftClientReply reply;
     try (RatisClient client = getRaftClient(newGroupConf)) {
-      reply =
-          client.getRaftClient().admin().setConfiguration(new ArrayList<>(newGroupConf.getPeers()));
-      if (!reply.isSuccess()) {
+      int basicWaitTime = 500;
+      int maxWaitTime = 10000;
```

Comment:
It feels like we can just check regularly here, without adding idempotent logic, because it doesn't make a big difference.

Reply:
This is also okay.